tests/fixtures.py

```diff
@@ -418,20 +412,23 @@ def _make_agency(testerchain, test_registry):
     token_deployer.deploy()
 
     staking_escrow_deployer = StakingEscrowDeployer(deployer_address=origin, registry=test_registry)
-    staking_escrow_deployer.deploy(secret_hash=os.urandom(DispatcherDeployer.DISPATCHER_SECRET_LENGTH))
+    staking_escrow_deployer.deploy(secret_hash=INSECURE_DEPLOYMENT_SECRET_HASH)
 
     policy_manager_deployer = PolicyManagerDeployer(deployer_address=origin, registry=test_registry)
-    policy_manager_deployer.deploy(secret_hash=os.urandom(DispatcherDeployer.DISPATCHER_SECRET_LENGTH))
+    policy_manager_deployer.deploy(secret_hash=INSECURE_DEPLOYMENT_SECRET_HASH)
 
     adjudicator_deployer = AdjudicatorDeployer(deployer_address=origin, registry=test_registry)
-    adjudicator_deployer.deploy(secret_hash=os.urandom(DispatcherDeployer.DISPATCHER_SECRET_LENGTH))
+    adjudicator_deployer.deploy(secret_hash=INSECURE_DEPLOYMENT_SECRET_HASH)
+
+    user_escrow_proxy_deployer = UserEscrowProxyDeployer(deployer_address=origin, registry=test_registry)
+    user_escrow_proxy_deployer.deploy(secret_hash=INSECURE_DEPLOYMENT_SECRET_HASH)
 
-    token_agent = token_deployer.make_agent()              # 1 Token
-    staking_agent = staking_escrow_deployer.make_agent()   # 2 Staking Escrow
-    policy_agent = policy_manager_deployer.make_agent()    # 3 Policy Agent
-    _adjudicator_agent = adjudicator_deployer.make_agent()  # 4 Adjudicator
+    token_agent = token_deployer.make_agent()                           # 1 Token
+    staking_agent = staking_escrow_deployer.make_agent()                # 2 Miner Escrow
```

Comment:
```suggestion
    staking_agent = staking_escrow_deployer.make_agent()                # 2 StakingEscrow
```

Reply:
:question:

Reply:
Go to the end of the line. There's a "Miner Escrow"

Reply:
:heavy_check_mark: